action.yml

```diff
@@ -155,20 +155,40 @@ runs:
         IPFS_CLUSTER_PASSWORD: ${{ inputs.cluster-password }}
       run: |
         echo "ℹ️ Uploading CAR with CID ${{ steps.merkleize.outputs.cid }} to IPFS Cluster" >> $GITHUB_STEP_SUMMARY
-        if [ -n "${ACTIONS_RUNNER_DEBUG}" ]; then
-          export GOLOG_LOG_LEVEL='debug'
-        fi
-        # the --local flag will add the CAR to the local IPFS daemon of the peer receiving the request, but not wait for it to be fully replicated.
-        ipfs-cluster-ctl \
-          --enc=json \
-          --host ${IPFS_CLUSTER_URL} \
-          --basic-auth ${IPFS_CLUSTER_USER}:${IPFS_CLUSTER_PASSWORD} \
-          add --format=car \
-          --local \
-          --name "${{ github.repository }}/${{ github.sha }}" \
-          build.car
-
-        echo "✅ Uploaded CAR with CID ${{ steps.merkleize.outputs.cid }} to IPFS Cluster" >> $GITHUB_STEP_SUMMARY
+
+        # run in a loop and retry
+        attempt=1
+        while true; do
+          echo "Attempt #$attempt"
+
+          # Make sure we don't loop forever
+          if [ $attempt -eq 5 ]; then
```

Comment:
nit: number of retries should be adjustable, or we should have `timeout-minutes: 15` set before `run`

Reply:
[timeout-minutes isn't available in composite actions](https://github.com/orgs/community/discussions/50481).

I'll make this adjustable